api/app.js

```diff
@@ -38,4 +38,6 @@ app.use(function(err, req, res, next) {
   res.render('error');
 });
 
+app.listen(5000, () => console.log('server listening...'))
```

Comment:
This is not necessary since you don't directly run the app.js. This whole express backend is just built with the express generator so all you need to do to run is call "npm start"

Reply:
Gotcha gotcha, can we talk about querying the backend on Wednesday